templates/qemu.conf

```diff
@@ -0,0 +1,5 @@
+<% if @qemu_conf -%>
+  <%- @qemu_conf.each do |key,value| -%>
```

Comment:
I think we should distinct for Array, String and Integers:

```
> <% if ( value.is_a? Array ) -%>
> <%= key %> = ["<%= value.join(", ") %>"]
> <% elseif  ( Integer(value) rescue false ) -%>
> <%= key %> = <%= value %>
> <% else -%> <%! String !%>
> <%= key %> = "<%= value %>"
> <% end -%>
> 
```
Like this we do not have to encode the format in the hash.

Reply:
You're right :)  Let me fix this

Reply:
Do you have some reference for the "<%! String !%>" format ? first time I see this